src/components/button-bar/index.js

```diff
@@ -34,7 +34,7 @@ const ButtonBarComponent = ({ step, changeStep, formValues, removeReservedTicket
                             {step !== 0 && step !== 2 && <button className={`${styles.button} button`} onClick={() => changeStep(step - 1)}>&lt; Back</button>}
                             {step !== 0 && step === 2 && <button className={`${styles.button} button`} onClick={() => removeReservedTicket()}>&lt; Back</button>}
```

Comment:
replace step with constants not number ( refactor this, constants are already defined)